lib/core/dns-resolver.js

```diff
@@ -0,0 +1,443 @@
+// source: https://raw.githubusercontent.com/szmarczak/cacheable-lookup/9e60c9f6e74a003692aec68f3ddad93afe613b8f/source/index.mjs
```

Comment:
please include the license at the top of the file

Reply:
yes no probs